src/xdocs/google_style.xml

```diff
@@ -2435,6 +2435,30 @@
           </table>
         </div>
       </subsection>
+      <subsection name="Overrides" id="Google_Overrides">
+        <p>
+          To change the default severity level, use the following custom property variable:
```

Comment:
Change:
````
<p>
It is possible to change the severity level of violations printed by using the Google style configuration provided by Checkstyle. To change the default severity level, set the system property <i>org.checkstyle.google.severity</i>.
</p>
<p>
To learn more about severity levels, please visit <a href="property_types.html#SeverityLevel">Severity Level</a>. To learn more about how to configure this property visit <a href="cmdline.html#Command_line_usage">CLI's <code>-p</code> option</a> , <a href="anttask.html#Parameters">Ant's <code>properties</code> parameter</a>, or use Java's system property before executing Checkstye.</p>
````

We really don't have documentation specific for system properties. This should be fixed in another issue.

Reply:
done, updated doc:

https://github.com/checkstyle/checkstyle/blob/9f162875898b7170f53693c13670e2206f7da912/src/xdocs/google_style.xml#L2435-L2456

Reply:
@romani @rnveach new section: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/49af32e_2024063236/google_style.html#Google_Severity_Level

Reply:
Remove the first 2 lines and then I am done.

````
To change the default severity level, use the following custom property variable:

org.checkstyle.google.severity=${SeverityLevel}
````

We should be explaining how to use this in other documentation and linking them to it. We don't really have this right now though.

Reply:
How about if we go with the following description?

----

To adjust the default severity level for violations using Checkstyle's Google style configuration, set the system property `org.checkstyle.google.severity=${SeverityLevel}`. This change will affect how violations are reported according to the severity level you specify.
 
For detailed guidance on severity levels, refer to the "Severity Level" (link) documentation. To configure this property, see the instructions for the command line interface's `-p` option (link), use the properties parameter in Ant (link), or set the Java system property before running Checkstyle.

----

Reply:
> set the system property org.checkstyle.google.severity=${SeverityLevel}. 

I avoiding this for 2 reasons.
1) I was trying to follow https://checkstyle.org/google_style.html#Google_Suppressions , which just talks about the name of the property but not how to set it and such. Really, our other documentation should be going into details like what is a severity and what are the options.
2) We are getting into implementation specific on how to set it and that just goes back tot he above. Yes, most places are `key=value` but as we found out when trying to test this, Windows needs quotes around it when using it with Java's command line.

Reply:
Agree, updated comment.

Reply:
I am good with @rdiachenko 's recommendation. It should replace the content in this entire section.